{
  "TOOLNAME": "py3-exceptiongroup",
  "TOOLREVISION": "1",
  "TOOLVERSION": "1.2.1-641f1198a5b180882b579b461d6953ba"
}